dds_web/api/project.py

```diff
@@ -124,6 +124,9 @@ def post(self):
         try:
             project.project_statuses.append(new_status_row)
             db.session.commit()
+        except (sqlalchemy.exc.OperationalError) as err:
```

Comment:
Suggestion:
```
        except (sqlalchemy.exc.OperationalError, sqlalchemy.exc.SQLAlchemyError) as err:
            flask.current_app.logger.exception(err)
            db.session.rollback()
            raise DatabaseError(message=str(err), alt_message="Database seems to be down." if isinstance(err, sqlalchemy.exc.OperationalError) else "Server Error: Status was not updated")
```

Reply:
is there a better way to apply the suggestion then copying the code in edit mode here or in the local file and then pushing/pulling

Reply:
Yeah there is, not when suggesting to overwrite code that hasn't been changed in the current PR, unfortunately..

Reply:
But if I wanted to suggest a change only for the lines you have added/changed here, you can write

```suggestion
        the suggested change here
```
![Screenshot 2022-03-24 at 15 51 32](https://user-images.githubusercontent.com/35953392/159943709-c087b717-c1ae-42df-a121-83f3cca6aca4.png)

or click this little page thing next to `H` and it automatically adds it to the comment
![Screenshot 2022-03-24 at 15 48 30](https://user-images.githubusercontent.com/35953392/159943793-a3cb0149-f302-4db5-a021-372be75ab851.png)

And then you can directly commit the changes